tools/st2_deploy.sh

```diff
@@ -123,10 +135,37 @@ if [[ "$DEBTEST" == "Ubuntu" ]]; then
 elif [[ -f "/etc/redhat-release" ]]; then
   TYPE="rpms"
   PYTHONPACK="/usr/lib/python2.7/site-packages"
+  if cat /etc/redhat-release | grep -q ' 6\.[0-9]'
+  then
+    OS_VER=6
+  elif cat /etc/redhat-release | grep -q ' 7\.[0-9]'
+  then
+    OS_VER=7
+  fi
   echo "###########################################################################################"
-  echo "# Detected linux distribution is RedHat compatible"
-  systemctl stop firewalld
-  systemctl disable firewalld
+  echo "# Detected linux distribution is RedHat ${OS_VER} compatible:"
+
+  if [ "$OS_VER" -eq "7" ]
+  then
+    systemctl stop firewalld
+    systemctl disable firewalld
+    PIP="pip"
+    VIRTUALENV="virtualenv"
+    PYTHON="python"
+    YUM_PYTHON=$(join " " ${YUM_PYTHON_7[@]})
+  elif [ "$OS_VER" -eq "6" ]
+  then
+    service firewalld stop
+    chkconfig firewalld off
+    PIP="pip2.7"
+    VIRTUALENV="virtualenv-2.7"
+    PYTHON="python2.7"
+    YUM_PYTHON=$(join " " ${YUM_PYTHON_6[@]})
+  else
+    echo "Unknown RHEL-family... Aborting install."
+    exit 2
+  fi
+
```

Comment:
Ooops, forgot about Fedora. This is easy to fix though. I am not really familiar with Fedora's versions and their python versions usage.